pkg/redis/RedisQueueConsumer.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Enqueue\Redis;
+
+class RedisQueueConsumer
```

Comment:
The name is misleading IMO. Could it be named `RedisInternalConsumer` or `RedisGenericConsumer`.